emodelrunner/create_cells.py

```diff
@@ -207,15 +205,15 @@ def get_precell(
 
     unopt_params_path = config.get("Paths", "precell_unoptimized_params_path")
 
-    morph_args = get_synplas_morph_args(config, precell=True)
+    morph = create_morphology(get_synplas_morph_args(config), config.package_type)
```

Comment:
This looks suspicious indeed. Let me dig more here.

Reply:
Since the protocol to make the precell fire is a strong short step, such that the cell fires one shortly after, maybe when using other morphologies, there is some chance that the cell still behaves about the same?
Also the pair simulation test threshold is a little bit loose, enabling not exact response to pass.
That might explain why the tests are passing with a different morphology

Reply:
Yes maybe that's the case. Which test is calling this function? Is it `test_synplas.py::test_pairsim_voltages`? 
The threshold is large there (1.0). Can we put a smaller value?

`check_output(threshold_v=1.0)`

Reply:
I added back the `precell=True` argument in 566838b. I must have been confused that with the package_type parameter.

Reply:
Yes, it's the `test_synplas.py::test_pairsim_voltages` that calls this function. We can use a smaller value for the threshold. The idea for the big threshold was that some cells produce traces that are close to the original one, but slightly different. When the peaks are offset, it can make the rms grow by a lot, thus the large threshold.

Reply:
The rms is actually `0.06496805725356769`.

Reply:
Yes, it always uses the same cell in the test. But I've set this threshold back when I was making tests with different cells.

Reply:
Ah yes I remember. Ok I set it to 0.07 in f197471